ui/src/app/testing/fake-auth.service.ts

```diff
@@ -0,0 +1,11 @@
+import {AuthService} from "../core/auth.service";
+import {CapabilitiesService} from "../core/capabilities.service";
+
+export class FakeAuthService extends AuthService {
```

Comment:
Unless this causes compilation issues otherwise, I probably wouldn't extend AuthService as it's probably making a bunch of remote calls.